rasgotransforms/rasgotransforms/transforms/metric/metric.yaml

```diff
@@ -0,0 +1,56 @@
+name: metric
+type: insight
+tags:
+  - insight
+description: Calculate metric values given a metric definition
+arguments:
+  timestamp:
+    type: column
+    description: The date/datetime column by which the metric will be aggregated
+  target_sql:
```

Comment:
`target_sql` can also be just a column name? maybe `target_expression`? or just `target`?

Reply:
Yeah I like target_expression better.